### ISSUE TITLE
[Request] Some VSync option like FIFO Relaxed

### ISSUE THREAD

**Issue by boludoz** [Ryujinx/Ryujinx#5649]
### Overview

I would like to have the possibility to choose FIFO relaxed as the vsync method among other options, that would increase the FPS abysmal at the cost of a bit of latency.

### Smaller details

FIFO Relaxed, Mailbox and more Vulkan options explained here: https://reddit.com/r/vulkan/s/rUI0W6T3p3

### Nature of request

Rquest

### Why would this feature be useful?

It would be useful for those who want low latency (Mailbox), something balanced (FIFO), or something fast (FIFO Relaxed).

**Comment by boludoz** [comment]
I am going to be honest, this emulator seems crazy to me, its new Experimental GUI is very beautiful, it is really very stable and of high quality, but it offers me few FPS, Yuzu has an option called FIFO Relaxed, that increases the stability of the FPS in an abysmal way, here it would be very useful for me at least.

**Comment by boludoz** [comment]
I also love that updates can be easily installed, the only thing that bothers people is the question of how difficult it is to apply mods or patches.

**Comment by Foul-Tarnished** [comment]
Latency is pretty bad as it is right now, but I guess it's just what happens when you emulate a modern GPU.

If the latency increase caused by FIFO Relaxed is small _in percentage_, that would be beneficial I guess

**Comment by boludoz** [comment]
The only thing I know is that Mailbox is slow.

**Comment by boludoz** [comment]
I have an RTX 3080 which won't be great but it's not the worst either and I really get very low FPS.

**Comment by boludoz** [comment]
If you want, I can try to implement these options, not all of us have the same tastes and needs.

**Comment by boludoz** [comment]
> In what way do you think mailbox is "slow"? Presentation mode will not singificantly impact framerate, it will only affect latency (and the framecap+tearing if immediate). If a game is running slowly, it's a combination of that game and your CPU. It's rarely the GPU.

Yes, my CPU is quite old, it is an i5 from 2017, but for some reason mailbox works badly compared to yuzu in relaxed mode.

**Comment by BParks21** [comment]
Please add these options 🙏
